integrations/streamer_test.go

```diff
@@ -3793,6 +3794,85 @@ stream
 	}
 }
 
+func TestStream_AlertTelegram(t *testing.T) {
+	requestCount := int32(0)
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		atomic.AddInt32(&requestCount, 1)
+		type postData struct {
+			ChatId                string `json:"chat_id"`
+			Text                  string `json:"text"`
+			ParseMode             string `json:"parse_mode"`
+			DisableWebPagePreview bool   `json:"disable_web_page_preview"`
+			DisableNotification   bool   `json:"disable_notification"`
+		}
+		pd := postData{}
+		dec := json.NewDecoder(r.Body)
+		dec.Decode(&pd)
+		if exp := "/botTOKEN:AUTH/sendMessage"; r.URL.String() != exp {
+			t.Errorf("unexpected url got %s exp %s", r.URL.String(), exp)
+		}
+		if exp := "12345678"; pd.ChatId != exp {
+			t.Errorf("unexpected recipient got %s exp %s", pd.ChatId, exp)
+		}
+		if exp := "HTML"; pd.ParseMode != exp {
+			t.Errorf("unexpected recipient got %s exp %s", pd.ParseMode, exp)
+		}
+		if exp := true; pd.DisableWebPagePreview != exp {
+			t.Errorf("unexpected DisableWebPagePreview got %t exp %t", pd.DisableWebPagePreview, exp)
+		}
+		if exp := true; pd.DisableNotification != exp {
+			t.Errorf("unexpected DisableNotification got %t exp %t", pd.DisableNotification, exp)
+		}
+		if exp := "kapacitor/cpu/serverA is CRITICAL"; pd.Text != exp {
+			t.Errorf("unexpected text got %s exp %s", pd.Text, exp)
+		}
+	}))
+	defer ts.Close()
+
+	var script = `
+stream
+	|from()
+		.measurement('cpu')
+		.where(lambda: "host" == 'serverA')
+		.groupBy('host')
+	|window()
+		.period(10s)
+		.every(10s)
+	|count('value')
+	|alert()
+		.id('kapacitor/{{ .Name }}/{{ index .Tags "host" }}')
+		.info(lambda: "count" > 6.0)
+		.warn(lambda: "count" > 7.0)
+		.crit(lambda: "count" > 8.0)
+		.telegram()
+		.chatId('12345678')
+                .disableNotification()
+                .parseMode('HTML')
```

Comment:
Can you add another `.telegram` call so we can show that it works with multiple handlers configured?